test/e2e/gpu/gpu.go

```diff
@@ -35,8 +35,10 @@ import (
 )
 
 const (
-	kustomizationYaml = "deployments/gpu_plugin/kustomization.yaml"
-	containerName     = "testcontainer"
+	kustomizationYaml   = "deployments/gpu_plugin/kustomization.yaml"
+	containerName       = "testcontainer"
+	tfKustomizationYaml = "demo/intel-gpu-tensorflow/kustomization.yaml"
```

Comment:
I think this could be under `deployments/` too to avoid the `Makefile` changes. I know it's not clear when something is under `demo/` but...

Reply:
Sure. I'll move it.